tests/test_structs.py

```diff
@@ -1041,17 +1041,20 @@ def test_lifted_decision_lists():
 
     atoms = {on_table([cup1]), hand_empty([robot])}
     goal = {on([cup1, plate1])}
+    objects = {o for a in atoms | goal for o in a.objects}
```

Comment:
so as to not suggest a bad pattern, let's just explicitly create the object set: `objects = {cup1, plate1, ...}`

Reply:
Good idea!